internal/cmd/commands/dev/dev.go

```diff
@@ -532,3 +539,27 @@ func (c *Command) Run(args []string) int {
 
 	return base.CommandSuccess
 }
+
+type oidcSetup struct {
+	testProvider                *capoidc.TestProvider
+	testProviderAlg             capoidc.Alg
+	testProviderCaCert          []*x509.Certificate
+	testController              *httptest.Server
+	authMethod                  *oidc.AuthMethod
+	testProviderAllowedRedirect string
+}
+
+func (c *Command) startOidcTestProvider() error {
+	/*
```

Comment:
for an example, see: https://github.com/hashicorp/cap/blob/935c659d2806460da893885239f1257d41d911fe/oidc/examples/cli/main.go#L125